lib/job.ts

```diff
@@ -47,7 +48,7 @@ export const upsertJob = async ({
   return job;
 };
 
-export const validJobRecords = async (records: Record[]) => {
```

Comment:
It's time to cut this function, the records we get back should be valid

Reply:
removed